### PR TITLE
Theme updates

### PR DESCRIPTION
**Pull request by Macroz**
- make the CSC theme look less like broken
- update the LBR and THL themes and related fixes (see also PR in `rems-deploy`)
- reduce default config to minimal for the theme and set dev-config to use example theme

See #890

### CODE REVIEW

File: src/clj/rems/css/styles.clj [hunk 1]
```diff
@@ -117,12 +126,15 @@
...
-                   :color "#fff"}]]])
+    [:&.active (merge {:color (util/get-theme-attribute :phase-color-active :phase-color "#000")}
+                      (if-let [background (util/get-theme-attribute :phase-background-active)]
+                        {:background background}
+                        {:background-color (util/get-theme-attribute :phase-bgcolor-active)
+                         :border-color (util/get-theme-attribute
+                                        :phase-bgcolor-active)}))]
+    [:&.completed {:background-color (util/get-theme-attribute :phase-bgcolor-completed)
+                   :border-color (util/get-theme-attribute :phase-bgcolor-completed)
+                   :color (util/get-theme-attribute :phase-color-completed :phase-color "#fff")}]]])
```
Comment (luontola) [inline src/clj/rems/css/styles.clj]: The fallback `"#fff"` is impossible to see on white background in the default theme. Using ` "#000"` as the fallback color for both active and completed phases would be more reliable.

File: example-theme/theme.edn [hunk 2]
```diff
@@ -16,9 +18,23 @@
...
+ :collapse-bgcolor "#f8f8f8"
+ :collapse-heading-color "#008278"
+ :table-bgcolor "#f8f8f8"
+ :table-heading-color "#fff"
+ :table-heading-bgcolor "#008278" ;"#006778" ;"#006fb1"; "#008278" "#eef2f8"
+ :table-hover-bgcolor "#b7dddb"; "#eef2f8"; "#c9e2ae" ; "#f2f2f2"
+ :table-text-color "#111"
+ :header-border "4px solid #ccc"
+ :table-shadow "10px -12px 21px -14px rgba(0,0,0,0.75)"
+ :link-color "#595959"
```
[the remark this answers is not in the document]
Comment (Macroz) [reply example-theme/theme.edn]: Used another variant (blue) from csc.fi for this that works better.

File: src/clj/rems/css/styles.clj [hunk 3]
```diff
@@ -190,16 +209,19 @@
...
            :height (u/percent 100)}]
-   [:body {:font-family "'Lato', sans-serif"
+   [:body {:font-family (util/get-theme-attribute :font-family "'Lato', sans-serif")
            :min-height (u/percent 100)
            :display :flex
            :flex-direction :column
            :padding-top (u/px 56)}]
+   [:h1 :h2 {:font-weight 500}]
+   [:h1 :h2 :h3 {:letter-spacing (u/rem 0.17)}]
+   [:h4 :h5 :h6 {:letter-spacing (u/rem 0.12)}]
```
[the remark this answers is not in the document]
Comment (Macroz) [reply src/clj/rems/css/styles.clj]: Changed it to regular `<span>`

File: resources/config-defaults.edn [hunk 4]
```diff
@@ -43,29 +43,10 @@
...
-         :phase-color "#cbd0d5"
-         :phase-color-active "#7eadba"
-         :phase-color-completed "#007887"
-         :table-bgcolor "#a4acb5"
-         :table-stripe-color "#cbd0d5"
-         :table-text-color "#fff"}
+ :theme {:color1 "#eee"
+         :color2 "#555"
+         :color3 "#000"
+         :color4 "#000"}
```
[the remark this answers is not in the document]
Comment (Macroz) [reply resources/config-defaults.edn]: I set the default for not finding specific `:link-color` to the Bootstrap default which is blue.

File: src/clj/rems/css/styles.clj (hunk 1 above)
Comment (Macroz) [reply src/clj/rems/css/styles.clj]: I changed the fallback colors and bgcolors to gray.

**Pull request merged**